### ISSUE TITLE
New netlify builds show up as "completed" but not "successful"

### ISSUE THREAD

**Issue by mdlincoln** [programminghistorian/jekyll#1604]
Just flagging for the team that I have seen the new Netlify build PR checks are showing up as "yellow" or incomplete when they really are complete. This is preventing editors from merging in approved PRs. I've put in a [help request to Netlify about this](https://community.netlify.com/t/github-actions-dont-report-successful/6739) and may try turning them on and off again so please just bear with me! In the meantime, I'll force-merge any approved and valid PRs.

**Comment by jenniferisasi** [comment]
Thanks @mdlincoln! I saw that on my PR for the newsletter and I had to stop myself from writing to you because vacation time.

**Comment by drjwbaker** [comment]
@mdlincoln: thanks for dealing with this during our close period. I hope this isn't related to migrating the account to the shared gmail..

**Comment by mdlincoln** [comment]
Still waiting on more advice from the Netlify folks. Please continue to follow the usual PR workflow though, and just ping me (or @drjwbaker since he can do this, too) to force merge the branch when ready.

**Comment by drjwbaker** [comment]
**I WILL ATTEMPT NOT TO BREAK THE SITE**

**Comment by mdlincoln** [comment]
OMFG I figured out a solution (if not a total and complete understanding of the problem.) Documenting it here as well as on the support ticket I logged with Netlify:

When we add any kind of integration to our repo (like netlify, or Travis, or heroku or w/e) it can send "status checks" based on our commits. GitHub keeps a list of all the kinds of notifications it receives under the ["Branch protection rules"](https://github.com/programminghistorian/jekyll/settings/branches) This is what lets a repo administrator like myself establish policies that prevent editors from merging a PR until the status checks are complete.

For whatever reason, Netlify sends two kinds of status checks. The `netlify/ph-preview/deploy` correctly reports status. The other, `netlify/deploy`, doesn't seem to send the "completed" notification to GitHub. I can't seem to fix that on the netlify side, but what I can do on our repo side is to set it so that only the `netlify/deploy` check (plus our Travis checks) are required for merging.

![Screen Shot 2020-01-08 at 9 22 54 AM](https://user-images.githubusercontent.com/4392604/71985664-ffb62900-31f8-11ea-916f-89af1ffeb7ed.png)
